CRM/Event/Form/Registration/Confirm.php

```diff
@@ -955,13 +953,7 @@ private function processContribution(
     }
 
     if (Civi::settings()->get('deferred_revenue_enabled')) {
-      $eventStartDate = CRM_Utils_Array::value(
```

Comment:
OK - I've clearly hit the glazes stage - had to stare at this for a bit to realise it is right